sources/mma/mma.f90

```diff
@@ -50,7 +50,11 @@ module mma
      private
 
      real(kind=rp) :: a0, f0val, asyinit, asyincr, asydecr, epsimin, &
-          residumax, residunorm
+          residumax
+     ! we need to have the KKT residual public
+     ! or alternatively, have KKT a function not a subroutine,
+     ! that return residunrom
+     real(kind=rp), public :: residunorm
```

Comment:
This is an internal variable and should never be accessed from the outside. Why did you need this?

Reply:
replaced with get_residunorm